app/config/parameters.yml.dist

```diff
@@ -6,7 +6,6 @@ parameters:
     secret: ThisTokenIsNotSoSecretChangeIt
     session_name: journal
     redis_cache: redis://localhost
-    redis_http_cache: '%redis_cache%'
```

Comment:
One Redis, things are cleared based on namespaces (so the DOI cache is cleared on a deploy, Guzzle cache isn't).